src/test/java/teammates/test/cases/common/StudentAttributesTest.java

```diff
@@ -196,119 +203,154 @@ public void testIsEnrollInfoSameAs(){
-        
+
+    @Test
+    public void testIsRegistered() throws Exception {
+        StudentAttributes sd = new StudentAttributes("sect 1", "team 1",
+                "name 1", "email@email.com", "comment 1", "course1");
+
         // Id is not given yet
         assertFalse(sd.isRegistered());
-        
+
         // Id empty
         sd.googleId = "";
         assertFalse(sd.isRegistered());
-        
+
         // Id given
         sd.googleId = "googleId.1";
         assertTrue(sd.isRegistered());
     }
-    
+
     @Test
-    public void testToString(){
-        StudentAttributes sd = new StudentAttributes("sect 1", "team 1", "name 1", "email@email.com", "comment 1", "course1");
-        assertEquals("Student:name 1[email@email.com]" + Const.EOL, sd.toString());
-        assertEquals("    Student:name 1[email@email.com]" + Const.EOL, sd.toString(4));
+    public void testToString() {
+        StudentAttributes sd = new StudentAttributes("sect 1", "team 1",
+                "name 1", "email@email.com", "comment 1", "course1");
+        assertEquals("Student:name 1[email@email.com]" + Const.EOL,
+                sd.toString());
+        assertEquals("    Student:name 1[email@email.com]" + Const.EOL,
+                sd.toString(4));
     }
-    
-    @Test 
-    public void testToEnrollmentString(){
-        StudentAttributes sd = new StudentAttributes("sect 1", "team 1", "name 1", "email@email.com", "comment 1", "course1");
-        assertEquals("sect 1|team 1|name 1|email@email.com|comment 1", sd.toEnrollmentString());
+
+    @Test
+    public void testToEnrollmentString() {
+        StudentAttributes sd = new StudentAttributes("sect 1", "team 1",
+                "name 1", "email@email.com", "comment 1", "course1");
+        assertEquals("sect 1|team 1|name 1|email@email.com|comment 1",
+                sd.toEnrollmentString());
     }
-    
+
     @Test
     public void testGetRegistrationLink() {
-        StudentAttributes sd = new StudentAttributes("sect 1", "team 1", "name 1", "email@email.com", "comment 1", "course1");
+        StudentAttributes sd = new StudentAttributes("sect 1", "team 1",
+                "name 1", "email@email.com", "comment 1", "course1");
         sd.key = "testkey";
-        String regUrl = new Url(Config.APP_URL + Const.ActionURIs.STUDENT_COURSE_JOIN_NEW)
-                                .withRegistrationKey(StringHelper.encrypt("testkey"))
-                                .withStudentEmail("email@email.com")
-                                .withCourseId("course1")
-                                .toString();
+        String regUrl = new Url(Config.APP_URL
+                + Const.ActionURIs.STUDENT_COURSE_JOIN_NEW)
+                .withRegistrationKey(StringHelper.encrypt("testkey"))
+                .withStudentEmail("email@email.com")
+                .withCourseId("course1")
+                .toString();
```

Comment:
Original is better. Try to align child elements to be on the right side of the parent element. Visual hierarchy goes left-to-right.